### ISSUE TITLE
E - Data feedback - Wireframe - how do we direct feedback to the right place?

### ISSUE THREAD

**Issue by James-Dargie** [jncc/marine-recorder#57]
We would be wise to ensure this is directed to the relevant domain custodian, so I think we need to ensure that the Survey title is not free-text, comes from a controlled list (searchable pop-up if required) and that the user chosen content is used to direct the query.

**Comment by GMDuncan** [comment]
Yes, very nice idea - taking under consideration/feasibility check!
